nucypher/policy/models.py

```diff
@@ -322,14 +308,14 @@ class TreasureMap:
                                   (bytes, KECCAK_DIGEST_LENGTH),  # hrac
                                   (UmbralMessageKit, VariableLengthBytestring)
                                   )
-    node_id_splitter = BytestringSplitter(PUBLIC_ADDRESS_LENGTH)
+    node_id_splitter = BytestringSplitter(int(PUBLIC_ADDRESS_LENGTH), Arrangement.ID_LENGTH)
 
-    class InvalidPublicSignature(Exception):
+    class InvalidSignature(Exception):
```

Comment:
:+1:

Reply:
Hmmm, thanks.  :-)

In the back of my head, I thought you might object to this change.  IE, if the publicly verifiable signature is invalid, do we want a *different* exception for that?

It turns out that, currently, our logic doesn't demand it, so I think it's best to go with this simpler version unless and until it does.

Reply:
No, I think a general `InvalidSignature` is great. I can see it needing to be moved somewhere else in the future, maybe, but it looks great!

Reply:
Yeah, awesome.  My thoughts exactly.  

At some point, it might make sense for this exception (whatever we end up calling it) and `VerifiableNode.InvalidNode` to inherit from the same base (currently, the later inherits `SuspiciousActivity`).

Reply:
Hmmm.... I could see that. I think I like that, but an `InvalidSignature` may not necessarily be suspicious activity _all the time_.